### PR TITLE
Bump provider infra to k8s-1.19

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
This PR perrforms some provider infra updates.

- **cluster, Change kubevirtci.sh to cluster.sh**
Currently the kubevirtci.sh uses an internal
var KUBEVIRTCI_PATH which collides with _kubevirtci
scripts.
In order to avoid this naming collision, we
change the name of the script to cluster.sh,
and it's variables and functions accordingly.

- **provider, bump provider to k8s-1.19**
Currently, we use a provider that deploys CNAO
with multus, which is needed for the cluster.
This provider is very old, so in order to replace
it we change to k8s-1.19 provider and also install
CNAO as part of make cluster-up procedure.

- **cluster scripts, Fix cluster.sh to remove _kubevirtci folder**
Currently, when kubevirtci tag version is changed, we get
an error when accessing kubevirtci::install.
Added a fix that will clean up the old _kubevirtci folder

**Special notes for your reviewer**:

### CODE REVIEW

File: cluster/up.sh [hunk 1]
```diff
@@ -2,10 +2,27 @@
 
 set -ex
 
-source ./cluster/kubevirtci.sh
-kubevirtci::install
+source ./cluster/cluster.sh
+cluster::install
 
-$(kubevirtci::path)/cluster-up/up.sh
+function getLatestCNAOPatchVersion {
```
Comment (qinqon) [inline cluster/up.sh:8]: Why don't we use the CNAO provider ? maybe we can just bump it.

**Review by qinqon — CHANGES_REQUESTED** [review]

**Comment by RamLavi** [comment]
/hold
let's make sure ci passes after CNAO is upgraded in kubevirtci

File: hack/cnao/cnao-cr.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1	
+kind: NetworkAddonsConfig	
+metadata:	
+  name: cluster	
+spec:	
+  imagePullPolicy: Always	
+  linuxBridge: {}	
+  multus: {}
```
Comment (maiqueb) [inline hack/cnao/cnao-cr.yaml:9]: Why are you installing linux bridge ? AFAIU, it isn't needed.

**Comment by maiqueb** [comment]
I might be saying something stupid, but don't we need to provision the CNAO CRD as well ? (afaiu, it fails because it does not know the CNAO related entities).

**Review by maiqueb — CHANGES_REQUESTED** [review]

**Comment by RamLavi** [comment]
> I might be saying something stupid, but don't we need to provision the CNAO CRD as well ? (afaiu, it fails because it does not know the CNAO related entities).

we recently [added](https://github.com/kubevirt/kubevirtci/pull/482) a new flag `KUBVIRT_WITH_CNAO_SKIP_CONFIG` to kubevirtci that allowd us to deploy the crd, namespace and operator.. but it may have some issues.. am checking

File: hack/cnao/cnao-cr.yaml (hunk 2 above)
Comment (RamLavi) [reply hack/cnao/cnao-cr.yaml:9]: my bad, will fix

**Comment by RamLavi** [comment]
@maiqueb after [kubevirtci-#487](https://github.com/kubevirt/kubevirtci/pull/487) get's merged and gets a new tag I'll update the PR and we should be good.

**Comment by maiqueb** [comment]
/lgtm

@phoracek

File: cluster/up.sh [hunk 3]
```diff
@@ -5,8 +5,14 @@ set -ex
 source ./cluster/cluster.sh
 cluster::install
 
+export KUBVIRT_WITH_CNAO_SKIP_CONFIG=true
```
Comment (phoracek) [inline cluster/up.sh:8]: KUBEVIRT. But I see it's a typo in kubevirtci itself, won't block on it.

**Review by phoracek — APPROVED** [review]
/lgtm
/approve

File: cluster/up.sh (hunk 3 above)
Comment (RamLavi) [reply cluster/up.sh:8]: damnn ><

**Pull request merged**